### ISSUE TITLE
Silent Login

### ISSUE THREAD

**Issue by drod21** [AxaFrance/oidc-client#837]
### Issue and Steps to Reproduce

Silent Login instructions are unclear - no clear route to use it.
redirect_uri cannot be used with silent_redirect_uri, but redirect_uri is required. 
Our application would like to use silent login instead of standard redirection.

### Versions
6.0.13

#### Expected
Documentation/explanation around silent login
### Additional Details

- Installed packages:

**Comment by guillaume-chervet** [comment]
Thank you for the issue @drod21 .

What is your need? I think you are already logged to your oidc provider and you want to connect the same user on new web application whithout any redirection. Is that rigth?

**Comment by drod21** [comment]
Yep! Exactly. I'm attempting to "fake" the no redirection and showing of different components, but typically i think this is handled through iframing in standard oidc flows.

Also, is there any plan on exposing the "logout" functionality  outside of the useOidc hook? or should i copy that code? I know this is outside the scope of the issue, but figured i'd rather ask the question than open a new issue.

basically, the use case is, we want to redirect users to logout on a 401 status code from our authenticated api's, and we do this in an axios interceptor

**Comment by guillaume-chervet** [comment]
For the logout, you can import the @axa-fr/react-oidc/vanilla/oidc and use the factory to retrieve yout current oidc instance configuration name. After that you will be able to call the logout.

Example of the component library https://github.com/AxaGuilDEv/react-oidc/blob/master/packages/react/src/oidc/ReactOidc.tsx

**Comment by guillaume-chervet** [comment]
I will add the silent login in few days.

**Comment by guillaume-chervet** [comment]
I just published 6.5.0-beta0 for testing your use case. 

I have to clean a litle bit the code in order to merge : https://github.com/AxaGuilDEv/react-oidc/pull/845

You can test the demo, Ihabe added a new button in MultiAuth page.
